airflow/settings.py

```diff
@@ -118,6 +118,27 @@
     "upstream_failed": "orange",
 }
 
+# list of the environment variables that need to be masked from the logs
+# some of these could be sensitive and there's no reason to log those
+ENV_VARIABLES_TO_MASK = [
+    "AIRFLOW__CORE__FERNET_KEY",
+    "AIRFLOW__SMTP__SMTP_PORT",
+    "AIRFLOW__SMTP__SMTP_PASSWORD",
+    "AIRFLOW__SMTP__SMTP_PASSWORD_SECRET",
```

Comment:
The `_secret` aren't sensitive either -- they are the name of a secret to fetch the creds from so aren't sensitive themselves.